tests/test_properties.py

```diff
@@ -40,6 +40,108 @@ def eval(self, x):
         return self.a * x + self.b
 
 
+def g(x):
+    """Non-linear model, Example 5.1 from Evensen 2019"""
+    x1 = x[0]
+    return x1 + 0.2 * x1**3
+
+
+@pytest.mark.parametrize("N", [100, 200])
+def test_that_projection_is_better_for_nonlinear_forward_model_big_N_small_p(N):
+    # For non-linear forward model, g, and m<N,
```

Comment:
Minor minor but perhaps user `p` in `m<N`.

Reply:
Thanks. I see Evensen 2019 employs `m`, so I guess I should change the naming of the test. 
![image](https://user-images.githubusercontent.com/13650516/229100615-2e4e1703-f143-421a-b3a1-c099665052f0.png)

Reply:
Evensen uses `m` for number of responses / observations I think. He uses `n` for number of parameters.

Reply:
You speak the truth, @dafeda

Reply:
Should be okay now
1632919
it now separates between `m` and `n`. 
Since both equals 1 in this special case, I went a bit fast and used `m` for `n`.